style.css

```diff
@@ -422,3 +426,151 @@ form {
   /* Internet Explorer 10-11 */
   color: black;
 }
+
+/* Hide desktop version element */
+.only-desktop {
+  display: none;
+}
+
+/* Use media queries to make my desktop version look like the templateIn figma
+(Desktop: 768px or wider). */
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] [Optional] All through the desktop version,  Please place some left, right margins to your design to match the figma design. it is part of the learning objectives for this milestone to understand how to parse a Figma design to create a UI. so, therefore, it is an important requirement that your design looks as much as possible to the Figma design 🙏